### PR TITLE
Examples

### PR DESCRIPTION
**Pull request by dbojicic**
- Draft examples added
- Updated How to read this guide page - added descriptions to Examples and Downloads pages

### CODE REVIEW

**Review by xtfer — APPROVED** [review]
These look fine to me, in principle. I have only vaguely skimmed over the XML, but I assume we are just trying to get something in place so we can keep going...?

**Comment by dt-r** [comment]
> These look fine to me, in principle. I have only vaguely skimmed over the XML, but I assume we are just trying to get something in place so we can keep going...?

@xtfer, exactly. There are a couple of examples (1 Path & 1 Imaging) to assist in demonstrating the IG. The co-chairs are added in as reviewers to be able to provide feedback to get them to a state where you are ok with them for the Ballot for Comment snapshot.

**Review by mjosborne1 — APPROVED** [review]
I think the resource examples on the whole look realistic. It's unlikely that the requisition id for the Pathology and Radiology request examples would be the same number and that comment also goes for the placer order number. I don't think this is a major concern for examples when not looked at in the context of a real ordering scenario.

File: input/pagecontent/index.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ This guide is divided into several pages which are listed at the top of each pag
 - [FHIR Artefacts](artifacts.html): These pages provide detailed descriptions and formal definitions for all the FHIR artefacts defined in this guide.
   - [Profiles and Extensions](profiles-and-extensions.html): This set of pages describes the profiles and extensions that are defined in this guide to exchange quality data. Each profile page includes a narrative description and guidance, formal definition and a "Quick Start" guide which summarises the supported search transactions for each profile. Although the guidance typically focuses on the profiled elements, it may also may focus on un-profiled elements to aid with implementation.
   - [Terminology](terminology.html): This set of pages lists the value sets and code systems defined in this guide.
-- [Examples](examples.html): This is a placeholder for when content becomes available.
-- [Downloads](downloads.html): This is a placeholder for when content becomes available.
+- [Examples](examples.html): This page lists all the examples used in this guide.
+- [Downloads](downloads.html): This page provides links to downloadable artefacts that developers can use to help them implement this guide.
```
Comment (dt-r) [inline input/pagecontent/index.md:36]: Need to either do the full change or revert this half improvement of the downloads page line and leave the fix to another PR that updates this fully in line with changes to page organisation from HL7 AU licensing.

File: input/examples/practitioner-guthridge-jarred.xml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+                <system value="http://terminology.hl7.org/CodeSystem/v2-0203"/>
+                <code value="NPI"/>
+                <display value="National provider identifier"/>
+            </coding>
+            <text value="HPI-I"/>
+        </type>
+        <system value="http://ns.electronichealth.net.au/id/hi/hpii/1.0"/>
+        <value value="8003611566718288"/>
+    </identifier>
+    <identifier>
```
Comment (dt-r) [inline input/examples/practitioner-guthridge-jarred.xml:26]: Remove Prescriber identifier - it isn't a Must Support identifier type.

File: input/examples/organization-elimbah-medical-center.xml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+<Organization xmlns="http://hl7.org/fhir">
+    <id value="elimbah-medical-center"/>
+    <extension url="http://hl7.org/fhir/StructureDefinition/resource-instance-name">
+        <valueString value="Organization - Elimbah Medical Center"/>
+    </extension>
+    <extension url="http://hl7.org/fhir/StructureDefinition/resource-instance-description">
+        <valueMarkdown value="Elimbah Medical Center"/>
+    </extension>
+    <meta>
+        <profile value="http://hl7.org.au/fhir/StructureDefinition/au-organization"/>
```
Comment (dt-r) [inline input/examples/organization-elimbah-medical-center.xml:12]: meta profile should be AU Core not AU Base.

File: input/examples/encounter-consultation.xml [hunk 4]
```diff
@@ -0,0 +1,47 @@
+<Encounter xmlns="http://hl7.org/fhir">
+    <id value="consultation"/>
+    <extension url="http://hl7.org/fhir/StructureDefinition/resource-instance-name">
+        <valueString value="Encounter - Consultation"/>
+    </extension>
+    <extension url="http://hl7.org/fhir/StructureDefinition/resource-instance-description">
+        <valueMarkdown value="Consultation with patient Fred Roberts"/>
+    </extension>
+    <meta>
```
Comment (dt-r) [inline input/examples/encounter-consultation.xml:9]: meta profile should be AU Core

**Review by dt-r — CHANGES_REQUESTED** [review]
See inline comments for requested changes.

File: input/examples/encounter-consultation.xml (hunk 4 above)
Comment (dbojicic) [reply input/examples/encounter-consultation.xml:9]: Done

File: input/examples/organization-elimbah-medical-center.xml (hunk 3 above)
Comment (dbojicic) [reply input/examples/organization-elimbah-medical-center.xml]: Done

File: input/examples/practitioner-guthridge-jarred.xml (hunk 2 above)
Comment (dbojicic) [reply input/examples/practitioner-guthridge-jarred.xml]: Done

File: input/pagecontent/index.md (hunk 1 above)
Comment (dbojicic) [reply input/pagecontent/index.md]: Reverted the change related to Downloads page

**Review by dt-r — APPROVED** [review]

**Comment by angusmillar** [comment]
Maybe too late. 
What benefit is gained by providing the IHI status (Active & Verified) if you don't send the IHI VerifiedDate?

**Comment by dt-r** [comment]
> Maybe too late. What benefit is gained by providing the IHI status (Active & Verified) if you don't send the IHI VerifiedDate?

@angusmillar, what's your preference? Add in IHI VerifiedDate or take out those status extensions in this set of examples?

**Comment by angusmillar** [comment]
@dt-r I'd add the date in.

**Comment by dbojicic** [comment]
Thanks @angusmillar and @dt-r, added IHI Verified Date to the Patient example.

**Comment by dbojicic** [comment]
Further changes made to the examples following the profile split to generic, Imaging and Pathology Request: updated meta.profile in ServiceRequest examples to indicate conformance to domain-specific profiles.

**Review by dt-r — APPROVED** [review]

**Comment by dt-r** [comment]
Further comments and fixes to these examples can be raised in GitHub.
